tests/vanilla/dependency.test.tsx

```diff
@@ -274,3 +274,43 @@ it('refreshes deps for each async read', async () => {
   resolve.splice(0).forEach((fn) => fn())
   expect(values).toEqual([0, 1])
 })
+
+it('handles errors in async atoms', async () => {
+  const errorAtom = atom(async () => {
+    throw new Error('Test error')
+  })
+
+  const store = createStore()
+
+  await expect(store.get(errorAtom)).rejects.toThrow('Test error')
+})
```

Comment:
This doesn't seem a dependency test. Maybe move to some other file?

Reply:
@dai-shi Appreciated for comments. You're absolutely right, it should be moved out of here. These changes have been removed from this pull request at this moment, after that, this pull request will be goal-focused enough and for dependency test only.